python-client/pyproject.toml

```diff
@@ -71,8 +72,9 @@ test = [
     "tensorflow-macos>=2.10.0, <2.11; sys_platform == 'darwin' and platform_machine == 'arm64'",
     "tensorflow>=2.10.0, <2.11; sys_platform != 'darwin' or platform_machine != 'arm64'",
     "tensorflow-text>=2.10.0, <2.11; platform_machine == 'x86_64' or platform_machine == 'amd64'",
-    "mlflow>2",
-    "wandb"
+    "mlflow>2",                                                                                    # Should it be somewhere else ?
```

Comment:
actually I don't know why we install a full mlflow in dev mode while mlflow-skinny being part of prod dependencies, maybe @rabah-khalek knows

Reply:
`mlflow.evaluate()` (where we implemented our plug-in) is not shipped with `mlflow-skinny`. It is needed for functional tests of the plugin.